<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>processing_quad_selector</class>
 <widget class="QWidget" name="processing_quad_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>22</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="main_layout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="cmbDemosaic">
     <property name="toolTip">
      <string>Demosaic</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbRetrieval">
     <property name="toolTip">
      <string>Phase Retrieval</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
     <property name="minimumContentsLength">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbProcessing">
     <property name="toolTip">
      <string>Post Processing</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
     <property name="minimumContentsLength">
      <number>6</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbDenoise">
     <property name="toolTip">
      <string>Denoise Mode</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
